SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
+
+## Supported Versions
+
+The OpenIndiana project commits to providing fixes or mitigations to each
+and every security vulnerabilities in software we deliver. Such a repositories
+of software sources are [illumos-gate](https://github.com/illumos/illumos-gate),
+[pkg5](https://github.com/OpenIndiana/pkg5), [slim_source](https://github.com/OpenIndiana/slim_source),
+and [time-slider](https://github.com/OpenIndiana/time-slider).
+
+Priority with which we handle security issues is based on severity of the issue.
+
+OpenIndiana is a rolling release distribution and has only one maintained branch
+called `hipster`. Other (historical) branches are not supported.
+
+| Version        | Supported |
+| ---------------|---------- |
```

Comment:
Just remove this table, as we presume no /dev installation should exist in a wild.